lada/models.py

```diff
@@ -249,6 +249,16 @@ def append_arbitrary_discard(self, fellows):
         db.session.commit()
 
 
+rodo = db.Table('rodo',
+                      db.Column('rodo_id', db.Integer, db.ForeignKey('rodos.id'), primary_key=True),
+                      db.Column('fellow_id', db.Integer, db.ForeignKey('fellow.id'), primary_key=True)
+                      )
+
+class Rodos(db.Model):
```

Comment:
to w zasadzie można by przerobić na Chirographs i w przyszłości wykorzystywać to na różnorakie zgody, nie tylko rodo